xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/locationPicker_macros.vm

```diff
@@ -65,18 +65,20 @@
       <label>$escapetool.xml($services.localization.render($options.preview.label))</label>
       <span class="xHint">$escapetool.xml($services.localization.render($options.preview.hint))</span>
     </dt>
-    <dd>
-      #if ($isDocumentTreeAvailable)
-        #documentPickerModal($options)
-        #locationPickerActions()
-      #end
+    <dd class="location-picker-preview">
```

Comment:
Hello! 
I tested out your solution and it's working properly on the default color theme (Iceberg). However, as you can see under there, it doesn't work with the Lumen color theme, that defines custom border properties to the `breadcrumb` class. 

Moreover, as for implementation, I can't just use the `breadcrumb` class on the `location-actions` because this class is used by a script to dynamically change the content of the breadcrumb depending on the form. That's why I added all properties of breadcrumb to a new css class using less (so the script wouldn't target the new block).

```
dd .location-actions {
  /* Retrieve style from the breadcrumb block. Needed for keyboard accessibility reorganization of the 
  location-picker component.*/
  .breadcrumb;
}
```


_Using Iceberg color theme_
![19140IcebergSplitBreadcrumbs](https://user-images.githubusercontent.com/28761965/216002416-248d344e-9158-4cb9-ab24-eab66a16a57a.png)


_Using Lumen color theme_
![19140LumenTestbreadcrumbssplit](https://user-images.githubusercontent.com/28761965/216001334-98f1f45c-a5cc-4199-9d1c-bbaad18d2592.png)

Reply:
Sorry, my previous comment is not relevant.
I added a couple rules for both of the children and this should be enough to correct the border issues (except if the border-radius gets changed itself in a color theme).
The commit to come solves those border problems, however there's still an addition to the `breadcrumb.less` file.

![19140LumenCorrected](https://user-images.githubusercontent.com/28761965/216015296-ca6a7bba-6a5c-4c33-a05d-4cc4bda541c5.png)